security/postmortem-template.md

```diff
@@ -0,0 +1,70 @@
+> Slimmed down template from: Betsy Beyer, Chris Jones, Jennifer Petoff, and Niall Richard
+> Murphy. [“Site Reliability
+> Engineering.”](https://landing.google.com/sre/book/chapters/postmortem.html),
+> modified from
+> https://raw.githubusercontent.com/dastergon/postmortem-templates/master/templates/postmortem-template-srebook.md.
+> Follow the SRE link for examples of how to populate.
+
+# Security postmortem for CVE-YEAR-ABCDEF
+
+## Date
+
+> YYYY-MM-DD
+
+## Authors
+
+> @foo, @bar, ...
+
+## Status
+
+> Draft | Final
+
+## Summary
+
+> A few sentence summary.
+
+## CVE issue(s)
+
+> https://github.com/envoyproxy/envoy/issues/${CVE_ISSUED_ID}
+
+## Root Causes
+
+> What defect in Envoy led to the CVEs? How did this defect arise?
+
+## Resolution
+
+> How was the security release process followed? How were the fix patches
+> structured and authored?
+
+## Detection
+
+> How was this discovered? Reported by XYZ, found by fuzzing? Private or public
+> disclosure?
+
+## Action Items
+
+> List the followup GH issues:
+> * [ ] [Title of issue](GH issue link)
```

Comment:
Is there a better way to do this in GH that doesn't involve manually copying titles and ticking checkboxes? Maybe we could link to a GH search for the action item issues, if we structure them with something searchable.

Reply:
If we decided to do things as an issue vs. a PR we could have TODO items with check boxes in the issue description and then actually see how many of them are done. So maybe that would make us lean towards using issues? I suppose with issues we could just label them with a "Post Mortem" label so they are easy to find. I don't feel strongly about it.

Reply:
I think if we do PR, we could go with the approach of linking to a GH search. The way this would work is you would open up a bunch of issues and include the tag "Action item for CVE-YYYY-ABCDEF". Then you create a permanent search string URL for this and it gets included in the PR. I kind of like this, since we don't have to independently maintain check boxes, we're adapting the postmortem format to native GH.

Reply:
SGTM